internal/pkg/cli/app_init.go

```diff
@@ -83,7 +83,10 @@ func (opts *InitAppOpts) Validate() error {
 		}
 	}
 	if opts.DockerfilePath != "" {
-		if _, err := listDockerfiles(opts.fs, opts.DockerfilePath); err != nil {
+		if !strings.HasSuffix(opts.DockerfilePath, "/Dockerfile") {
```

Comment:
What do you think about only checking if the path is a directory and erroring out then? If it's a file we just pass.

Right now, I think there can be two things that are problematic. 
1. Users can have Dockerfiles that don't end with "Dockerfile" (https://docs.docker.com/engine/reference/commandline/build/#specify-a-dockerfile--f) so we should be able to handle those.
2. I'm not sure if this would work with windows because of the forward slash.

Reply:
Oh and can we update the flag description to "Path to the Dockerfile." instead of "Path to the directory with Dockerfile."

Reply:
:+1: Addressed